include/macros.h

```diff
@@ -3,6 +3,9 @@
 
 #include "convert.h"
 
+#define true 1
```

Comment:
These shouldn't go in `macros.h`. Instead, they should in `stdbool.h`. Copy what I have in #85